content/using/install.md

```diff
@@ -225,65 +236,106 @@ Find the absolute path to the keyfile that you downloaded from Bridge. Copy it.
 
 Never boot multiple instances of your ship at the same time. You can prevent this from happening on accident by only ever keeping a single copy of your pier.
 
-**Important:** once a key has been used to boot a ship onto the network, it cannot be used to boot that ship again later - doing so will cause communication problems with other ships. For this reason you should **delete the keyfile from your machine once your ship has booted successfully**. (If you do use the same key twice, you'll need to conduct a [personal breach](#breaches) to restore your ship to full functionality).
+**Important:** once a key has been used to boot a ship onto the network, it cannot be used to boot that ship again later - doing so will cause communication problems with other ships. 
+
+For this reason you should **delete the keyfile from your machine once your ship has booted successfully**. 
+
+If you do use the same key twice, you'll need to conduct a [personal breach](#breaches) to restore your ship to full functionality). 
+
+Delete the keyfile with the command below:
+
+```sh
+rm path/to/my-planet.key
+```
 
 ### Updating to the latest binary
 
 Most updates to Urbit are downloaded and applied automatically as OTA (Over the Air) updates. Occasionally it would be infeasible to distribute an update this way, and a new `urbit` binary is released. This is announced in the [urbit-dev](https://groups.google.com/a/urbit.org/forum/#!forum/dev) Google Group when it occurs (as are all OTA updates). 
 
 To update to the latest binary, download and extract it to replace your existing one. Then run it as before.
 
-For example, if in your current `urbit` directory you have an `urbit-v0.10.7-darwin` directory along with a ship directory called `mycomet`, and a new `urbit-v0.10.8-darwin` binary was just released, then you'd run the following commands from your `urbit` directory (the example commands below use the darwin installer, but you should use the one for your OS):
+For example, if in your current `urbit` directory you have an `urbit-v0.10.7-<OS>` directory along with a ship directory called `mycomet`, and a new `urbit-v0.10.8-<OS>` binary was just released, then you'd run the following commands from your `urbit` directory:
 
-First shut down your ship, by running the following in dojo (or control-d).
+First shut down your ship, by running the following in dojo (or `control-d`).
 ```
 |exit
 ```
 
-Next download and extract the most recent binary from inside your `urbit` directory:
+Delete the old binary directory from your `urbit` directory:
+
+**NOTE**: Your ship itself can be reused, only the binary needs to be changed. This is why we have the directory structure configured as an `urbit` directory that contains your ship and your binary. This structure makes it easy to swap in new binaries. If you have your ship *inside* the `urbit-vX.XX.X` directory, move it out before deleting that directory.
```

Comment:
A little verbose. We're about to demonstrate that it's the same ship on line 327

Reply:
I'm worried people who have previously put their ship in their binary directory might delete it if they don't take it out first.

So it's a little verbose, but I thought explicitly stating ships can be reused and to not delete your old one would be good.